### DISCUSSION TITLE
Tooltip overflows in Datatable

### DISCUSSION THREAD

**Discussion by reinisg** [carbon-design-system/carbon#12812] [Help] [answered]
Hey, 
Can you please help resolve problem with Tooltips inside DataTable. Wanted to know is there any solution for this. Made test case in [CODESANBOX](https://codesandbox.io/s/sample-data-table-forked-4vdq2j?file=/index.js) If hover last table row, tooltip is invisible, how I could resolve?
Only for solution what could help - if condition for last row tooltips use alignment to top. Is there any other solutions?

**Comment by mbgower** [comment, marked as answer]
The tooltip is there (you can see the  vertical scrollbar to extend when it's activated). You have to scroll down to see it. 
Obviously not ideal.

You've specified a bottom alignment for your tips. `<Tooltip align="bottom" label="Tasty">`
If you remove your align attribute entirely, the tips will automatically position. Here's your bottom row with no position specified
<img width="727" alt="Screenshot 2023-01-04 at 7 40 47 AM" src="https://user-images.githubusercontent.com/14298245/210593011-a07da49a-3598-4c19-88b3-b1452fca0d07.png">

 All the tooltips are visible without the align attribute set (although the one on the top row is slightly truncated at the top).

**Reply by reinisg** [reply]
Thanks for answer!
In my case i did  this aligments for tooltips what are on table side and be overflowed. :)
